### PR TITLE
fix: fix missing streams errors 

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

**Review by jyeshe — CHANGES_REQUESTED** [review]
Solved the test case that was not passing and refactoring looks good.
Only one detail on error handling.

File: lib/ae_mdw/blocks.ex [hunk 1]
```diff
@@ -73,6 +73,19 @@ defmodule AeMdw.Blocks do
     hash
   end
 
+  @spec fetch_txis_from_gen(height()) :: Enumerable.t()
+  def fetch_txis_from_gen(height) do
+    with Model.block(tx_index: tx_index_start)
+         when is_integer(tx_index_start) and tx_index_start >= 0 <-
+           Database.fetch!(@table, {height, -1}),
```
Comment (jyeshe) [inline lib/ae_mdw/blocks.ex:80]: part of the 500 trap

**Review by thepiwo — APPROVED** [review]

**Review by jyeshe — APPROVED** [review]
Approving as I need this fix to conclude the rocksdb PR. The 500 happens only when the height is not yet present on Mdw so we might have a minor fix on a following PR if you please.

**Comment by sborrazas** [comment]
@jyeshe fixed, thank you. Merging now

**Pull request merged**